### PR TITLE
Booting from USB device leads to mounting error #527

### PR DESCRIPTION
**Pull request by joda01**
This is just a quick and dirty suggestion to solve the problem described in #527. Feel free to edit the solution and suggest a better implementation.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -214,6 +214,12 @@ get_iso()
                 break
             fi
         done
+    else
```
Comment (liyimeng) [inline install.sh:217]: This is not enough if there happen to have another disk mountable but actually contains no k3os, right?

Comment (OnnoH) [reply install.sh:217]: Not true. The else is fired when there actually is a k3os labeled partition. It just removes the number (/dev/sdb3 to /dev/sdb in my case), so the mounting further down the line works.

**Comment by sb3rg** [comment]
Having the same issue with USB boot on bare metal.  Trying to install yields in the same mounting error described in #527 .   I'm using Intel (Z400 desktop), BIOS based Motherboard.

 Attempted updating the k3os_install.sh file:  "...k3os-amd64\k3os\system\k3os\v0.11.0-rc1\k3os-install.sh" as suggested by editing k3os-amd64.iso and recreating image, with ImgBurn tool.  Upon booting from burned new image, grub boot menu appears with the three load options, but any one I select leaves me with:

> error: no such device:  K3OS
> error: file '//k3os/system/kernel/current/kernel.squashfs' not found
> error: disk 'loop0' not found.
> error: you need to load the kernel first.

I'm not a programmer by trade, just a hobbyist trying to create my own kubernetes cluster and failing miserably at every turn.  k3os shows promise.  I was hoping it was as easy as booting bare metal like rancherOS, but that seems not to be the case.  Hope I can get some help on this!  Thank you!

**Comment by OnnoH** [comment]
@sb3rg Sean, I am a programmer, and know something about hardware, and a little bit of all other fields in IT. That certainly is no guarantee for succes :-D

I have the same goal as you, throwing some Raspberry Pies in the mix as well. @joda01 Joachim's patch helped a lot!

Based on your error, I would say something went wrong during the burn process. Since you're using ImgBurn I guess you use MS Windows. ImgBurn seems quite dated. On my Mac I use Etcher (https://www.balena.io/etcher/) which also has a Windows flavour. Perhaps you can give that a go.

By the way: I built K3OS from source and burned the result. I have no experience with remastering the ISO, but that should work also.

Hang in there! In the end we shall prevail ;-)

**Comment by Kampe** [comment]
@dweomer has mentioned in k3os Slack that this fix will be brought in next release, not this whole commit as there's some other artifacts along for the ride, but at least the device logic anyway.

**Comment by joda01** [comment]
Thanks a lot for your feedback and testing! I will remove the other artifacts to make the Pull Request clean and ready for merge.

**Comment by joda01** [comment]
@Kampe I have removed the other artifacts and reduced the merge request just to the proposed fix.

**Comment by dweomer** [comment]
Superseded by #553

**Pull request closed without merging**